extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -338,6 +339,17 @@ export class SubscriptionService {
     });
   }
 
+  /**
+   * Logs error and sends message to viewer
+   * @param {string} error
+   */
+  sendAuthTokenErrorToViewer_(error) {
+    this.viewer_.sendMessageAwaitResponse('auth-rejected', dict({
+      'reason': error,
```

Comment:
You can't pass an `Error` object - must be a string.

Reply:
just a bad name, it actually is a string